command-expansion-server/src/app.ts

```diff
@@ -582,7 +582,15 @@ app.post('/get-seqjson-for-seqid-and-simulation-dataset', async (req, res, next)
 // General error handler
 app.use((err: any, _: Request, res: Response, next: NextFunction) => {
   logger.error(err);
-  res.status(err.status ?? err.statusCode ?? 500).send({ message: err.message });
+
+  res.status(err.status ?? err.statusCode ?? 500).send({
+    message: err.message,
+    extensions: {
+      ...(err.cause ? { cause: err.cause } : {}),
+      ...(err.stack ? { cause: err.stack } : {}),
```

Comment:
I think the `stack` case needs to be `{ stack: err.stack }`. Otherwise it will overwrite `cause` if both `stack` and `cause` exist.

Reply:
good catch.

Reply:
I made the change